UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -471,6 +473,8 @@
   gEfiMdeModulePkgTokenSpaceGuid.PcdVpdBaseAddress|0x0
   gEfiMdeModulePkgTokenSpaceGuid.PcdStatusCodeUseMemory|FALSE
   gEfiMdeModulePkgTokenSpaceGuid.PcdUse1GPageTable|TRUE
+  gUefiPayloadPkgTokenSpaceGuid.PcdHandOffFdtEnable|TRUE

```

Comment:
Set PcdHandOffFdtEnable to FALSE as default for now, platforms ready to use FDT will toggle this PCD as part of the UPL build command (--pcd NAME=VALUE)

Reply:
resolved